2020-09-24T12:08:12.644+0800 I CONTROL  [initandlisten] MongoDB starting : pid=394 port=27017 dbpath=/data/db 64-bit host=CN-00015440
2020-09-24T12:08:12.644+0800 I CONTROL  [initandlisten] db version v3.6.3
2020-09-24T12:08:12.644+0800 I CONTROL  [initandlisten] git version: 9586e557d54ef70f9ca4b43c26892cd55257e1a5
2020-09-24T12:08:12.644+0800 I CONTROL  [initandlisten] OpenSSL version: OpenSSL 1.1.1  11 Sep 2018
2020-09-24T12:08:12.644+0800 I CONTROL  [initandlisten] allocator: tcmalloc
2020-09-24T12:08:12.644+0800 I CONTROL  [initandlisten] modules: none
2020-09-24T12:08:12.644+0800 I CONTROL  [initandlisten] build environment:
2020-09-24T12:08:12.645+0800 I CONTROL  [initandlisten]     distarch: x86_64
2020-09-24T12:08:12.645+0800 I CONTROL  [initandlisten]     target_arch: x86_64
2020-09-24T12:08:12.645+0800 I CONTROL  [initandlisten] options: {}
2020-09-24T12:08:12.645+0800 W -        [initandlisten] Detected unclean shutdown - /data/db/mongod.lock is not empty.
2020-09-24T12:08:12.649+0800 I -        [initandlisten] Detected data files in /data/db created by the 'wiredTiger' storage engine, so setting the active storage engine to 'wiredTiger'.
2020-09-24T12:08:12.650+0800 W STORAGE  [initandlisten] Recovering data from the last clean checkpoint.
2020-09-24T12:08:12.650+0800 I STORAGE  [initandlisten] wiredtiger_open config: create,cache_size=15793M,session_max=20000,eviction=(threads_min=4,threads_max=4),config_base=false,statistics=(fast),log=(enabled=true,archive=true,path=journal,compressor=snappy),file_manager=(close_idle_time=100000),statistics_log=(wait=0),verbose=(recovery_progress),
2020-09-24T12:08:13.371+0800 I STORAGE  [initandlisten] WiredTiger message [1600920493:371679][394:0x7fff654b10c0], txn-recover: Main recovery loop: starting at 8/256
2020-09-24T12:08:13.382+0800 I STORAGE  [initandlisten] WiredTiger message [1600920493:382741][394:0x7fff654b10c0], txn-recover: Recovering log 8 through 9
2020-09-24T12:08:13.474+0800 I STORAGE  [initandlisten] WiredTiger message [1600920493:474504][394:0x7fff654b10c0], file:index-3-5664233315247082120.wt, txn-recover: Recovering log 9 through 9
2020-09-24T12:08:13.609+0800 I CONTROL  [initandlisten] 
2020-09-24T12:08:13.610+0800 I CONTROL  [initandlisten] ** WARNING: Access control is not enabled for the database.
2020-09-24T12:08:13.610+0800 I CONTROL  [initandlisten] **          Read and write access to data and configuration is unrestricted.
2020-09-24T12:08:13.610+0800 I CONTROL  [initandlisten] ** WARNING: You are running this process as the root user, which is not recommended.
2020-09-24T12:08:13.610+0800 I CONTROL  [initandlisten] 
2020-09-24T12:08:13.610+0800 I CONTROL  [initandlisten] ** WARNING: This server is bound to localhost.
2020-09-24T12:08:13.610+0800 I CONTROL  [initandlisten] **          Remote systems will be unable to connect to this server. 
2020-09-24T12:08:13.610+0800 I CONTROL  [initandlisten] **          Start the server with --bind_ip <address> to specify which IP 
2020-09-24T12:08:13.610+0800 I CONTROL  [initandlisten] **          addresses it should serve responses from, or with --bind_ip_all to
2020-09-24T12:08:13.610+0800 I CONTROL  [initandlisten] **          bind to all interfaces. If this behavior is desired, start the
2020-09-24T12:08:13.610+0800 I CONTROL  [initandlisten] **          server with --bind_ip 127.0.0.1 to disable this warning.
2020-09-24T12:08:13.610+0800 I CONTROL  [initandlisten] 
2020-09-24T12:08:13.629+0800 I FTDC     [initandlisten] Initializing full-time diagnostic data capture with directory '/data/db/diagnostic.data'
2020-09-24T12:08:13.631+0800 I NETWORK  [initandlisten] waiting for connections on port 27017
2020-09-24T12:08:14.026+0800 I FTDC     [ftdc] Unclean full-time diagnostic data capture shutdown detected, found interim file, some metrics may have been lost. OK
2020-09-24T12:08:16.929+0800 I NETWORK  [listener] connection accepted from 127.0.0.1:51911 #1 (1 connection now open)
2020-09-24T12:08:16.934+0800 I NETWORK  [conn1] received client metadata from 127.0.0.1:51911 conn: { driver: { name: "nodejs", version: "3.1.3" }, os: { type: "Linux", name: "linux", architecture: "x64", version: "4.4.0-17763-Microsoft" }, platform: "Node.js v8.10.0, LE, mongodb-core: 3.1.2" }
2020-09-24T12:08:22.018+0800 I NETWORK  [listener] connection accepted from 127.0.0.1:51921 #2 (2 connections now open)
2020-09-24T12:08:22.024+0800 I NETWORK  [conn2] received client metadata from 127.0.0.1:51921 conn: { driver: { name: "nodejs", version: "3.1.3" }, os: { type: "Linux", name: "linux", architecture: "x64", version: "4.4.0-17763-Microsoft" }, platform: "Node.js v8.10.0, LE, mongodb-core: 3.1.2" }
2020-09-24T12:09:45.595+0800 I CONTROL  [initandlisten] MongoDB starting : pid=508 port=27017 dbpath=/data/db 64-bit host=CN-00015440
2020-09-24T12:09:45.596+0800 I CONTROL  [initandlisten] db version v3.6.3
2020-09-24T12:09:45.596+0800 I CONTROL  [initandlisten] git version: 9586e557d54ef70f9ca4b43c26892cd55257e1a5
2020-09-24T12:09:45.596+0800 I CONTROL  [initandlisten] OpenSSL version: OpenSSL 1.1.1  11 Sep 2018
2020-09-24T12:09:45.596+0800 I CONTROL  [initandlisten] allocator: tcmalloc
2020-09-24T12:09:45.596+0800 I CONTROL  [initandlisten] modules: none
2020-09-24T12:09:45.596+0800 I CONTROL  [initandlisten] build environment:
2020-09-24T12:09:45.596+0800 I CONTROL  [initandlisten]     distarch: x86_64
2020-09-24T12:09:45.596+0800 I CONTROL  [initandlisten]     target_arch: x86_64
2020-09-24T12:09:45.596+0800 I CONTROL  [initandlisten] options: {}
2020-09-24T12:09:45.597+0800 W -        [initandlisten] Detected unclean shutdown - /data/db/mongod.lock is not empty.
2020-09-24T12:09:45.600+0800 I -        [initandlisten] Detected data files in /data/db created by the 'wiredTiger' storage engine, so setting the active storage engine to 'wiredTiger'.
2020-09-24T12:09:45.600+0800 W STORAGE  [initandlisten] Recovering data from the last clean checkpoint.
2020-09-24T12:09:45.600+0800 I STORAGE  [initandlisten] wiredtiger_open config: create,cache_size=15793M,session_max=20000,eviction=(threads_min=4,threads_max=4),config_base=false,statistics=(fast),log=(enabled=true,archive=true,path=journal,compressor=snappy),file_manager=(close_idle_time=100000),statistics_log=(wait=0),verbose=(recovery_progress),
2020-09-24T12:09:46.299+0800 I STORAGE  [initandlisten] WiredTiger message [1600920586:299365][508:0x7f4752cb10c0], txn-recover: Main recovery loop: starting at 9/3328
2020-09-24T12:09:46.308+0800 I STORAGE  [initandlisten] WiredTiger message [1600920586:308845][508:0x7f4752cb10c0], txn-recover: Recovering log 9 through 10
2020-09-24T12:09:46.393+0800 I STORAGE  [initandlisten] WiredTiger message [1600920586:393449][508:0x7f4752cb10c0], txn-recover: Recovering log 10 through 10
2020-09-24T12:09:46.528+0800 I CONTROL  [initandlisten] 
2020-09-24T12:09:46.528+0800 I CONTROL  [initandlisten] ** WARNING: Access control is not enabled for the database.
2020-09-24T12:09:46.528+0800 I CONTROL  [initandlisten] **          Read and write access to data and configuration is unrestricted.
2020-09-24T12:09:46.528+0800 I CONTROL  [initandlisten] ** WARNING: You are running this process as the root user, which is not recommended.
2020-09-24T12:09:46.528+0800 I CONTROL  [initandlisten] 
2020-09-24T12:09:46.528+0800 I CONTROL  [initandlisten] ** WARNING: This server is bound to localhost.
2020-09-24T12:09:46.528+0800 I CONTROL  [initandlisten] **          Remote systems will be unable to connect to this server. 
2020-09-24T12:09:46.528+0800 I CONTROL  [initandlisten] **          Start the server with --bind_ip <address> to specify which IP 
2020-09-24T12:09:46.528+0800 I CONTROL  [initandlisten] **          addresses it should serve responses from, or with --bind_ip_all to
2020-09-24T12:09:46.528+0800 I CONTROL  [initandlisten] **          bind to all interfaces. If this behavior is desired, start the
2020-09-24T12:09:46.529+0800 I CONTROL  [initandlisten] **          server with --bind_ip 127.0.0.1 to disable this warning.
2020-09-24T12:09:46.529+0800 I CONTROL  [initandlisten] 
2020-09-24T12:09:46.552+0800 I FTDC     [initandlisten] Initializing full-time diagnostic data capture with directory '/data/db/diagnostic.data'
2020-09-24T12:09:46.554+0800 I NETWORK  [initandlisten] waiting for connections on port 27017
2020-09-24T12:09:47.009+0800 I FTDC     [ftdc] Unclean full-time diagnostic data capture shutdown detected, found interim file, some metrics may have been lost. OK
2020-09-24T12:09:49.834+0800 I NETWORK  [listener] connection accepted from 127.0.0.1:52092 #1 (1 connection now open)
2020-09-24T12:09:49.840+0800 I NETWORK  [conn1] received client metadata from 127.0.0.1:52092 conn: { driver: { name: "nodejs", version: "3.1.3" }, os: { type: "Linux", name: "linux", architecture: "x64", version: "4.4.0-17763-Microsoft" }, platform: "Node.js v8.10.0, LE, mongodb-core: 3.1.2" }
2020-09-24T12:09:54.515+0800 I NETWORK  [listener] connection accepted from 127.0.0.1:52110 #2 (2 connections now open)
2020-09-24T12:09:54.521+0800 I NETWORK  [conn2] received client metadata from 127.0.0.1:52110 conn: { driver: { name: "nodejs", version: "3.1.3" }, os: { type: "Linux", name: "linux", architecture: "x64", version: "4.4.0-17763-Microsoft" }, platform: "Node.js v8.10.0, LE, mongodb-core: 3.1.2" }
2020-09-24T12:22:20.383+0800 I NETWORK  [listener] connection accepted from 127.0.0.1:53015 #3 (3 connections now open)
